app/seeders/dinosaur_seeder.rb

```diff
@@ -0,0 +1,28 @@
+class DinosaurSeeder
+  DINOSAURS = [
+    {
+        name: "Giganotosaurus",
+        location_found: "South America",
+        info_url: "https://en.wikipedia.org/wiki/Giganotosaurus",
+        user: User.first,
+        description: "This dinosaur is giganotic!"
+      }, {
+        name: "Spinosaurus",
+        location_found: "North Africa",
+        info_url: "https://en.wikipedia.org/wiki/Spinosaurus",
+        user: User.first,
+        description: "This dinosaur is spiny!"
+      }, {
+        name: "Tyrannosaurus",
+        location_found: "Western North America",
+        info_url: "https://en.wikipedia.org/wiki/Tyrannosaurus",
+        user: User.last,
+        description: "This dinosaur has itty bitty arms!"
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.